core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/java/MavenCoordinates.java

```diff
@@ -70,7 +81,7 @@ public boolean isEmpty() {
 
     @Override
     public boolean matches(ArtifactIdentifier artifactIdentifier) {
-        if(artifactIdentifier instanceof MavenCoordinates) {
+        if (artifactIdentifier instanceof MavenCoordinates) {
```

Comment:
Nit: Strictly speaking, the unrelated whitespace changes here and below should be moved out to a separate commit.

Reply:
Unfortunately a result of reformatting the file after my changes above this. Really removing that from the file?

Reply:
Well, I leave it to how strictly you handle this (hence only a "nit"). In ORT, we don't allow to mix topics within a single commit, and that includes formatting, as we try to separate functional from non-functional changes for more speedy reviews.

Reply:
FWIW, I didn't mean to *drop* the change from this PR, but just to separate it out into another commit in *this* PR.

Reply:
These changes were not intended at all, they were result of auto formatting, so I discarded them. In general, perhaps we should define at some point a common formatting and apply this over time to the source base. But this is not part of this change :-)